scripts/test_images.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+# Exit on error
+set -e
+
+# Consider command as failed when any component of the pipe fails:
+# https://stackoverflow.com/questions/1221833/pipe-output-and-capture-exit-status-in-bash
+set -o pipefail
```

Comment:
Please update this per https://github.com/etcd-io/etcd/issues/15514

Reply:
When I updated the script to strict mode, it is failing on "unbound variable (-u)" in `test/build_lib.sh` which is being addressed by 2 PRs (https://github.com/etcd-io/etcd/pull/15519 , https://github.com/etcd-io/etcd/pull/15513) currently. I can wait for it to be merged before testing/squashing. 

cc: @serathius @fuweid @chaochn47

Reply:
https://github.com/etcd-io/etcd/pull/15519 is already merged. Please rebase this PR and resolve the conflict.